Objects/longobject.c

```diff
@@ -5260,6 +5260,31 @@ long_is_finite(PyObject *v)
 }
 #endif
 
+/*[clinic input]
+int.as_integer_ratio
+
+Return integer ratio.
+
+Return a pair of integers, whose ratio is exactly equal to the original int
+and with a positive denominator.
+
+>>> (10).as_integer_ratio()
+(10, 1)
+>>> (0).as_integer_ratio()
+(0, 1)
+>>> (11).as_integer_ratio()
+(11, 1)
+>>> (-10).as_integer_ratio()
+(-10, 1)
+[clinic start generated code]*/
+
+static PyObject *
+int_as_integer_ratio_impl(PyObject *self)
+/*[clinic end generated code: output=e60803ae1cc8621a input=ce9c7768a1287fb9]*/
+{
+  return PyTuple_Pack(2, self, _PyLong_One)
```

Comment:
There's a missing semicolon at the end of this line; I think that's why the continuous integration builds are failing.

Also, please change the indentation of this line to 4 spaces instead of 2 spaces, following [PEP7](https://www.python.org/dev/peps/pep-0007/#code-lay-out).  Thanks!